themes/default/content/registry/packages/lbrlabs-eks/_index.md

```diff
@@ -4,24 +4,430 @@ meta_desc: Provides an overview of the batteries include EKS provider for Pulumi
 layout: package
 ---
 
+<!-- markdownlint-disable MD034 -->
```

Comment:
@jaxxstorm - this PR was failing linting because of the bare urls that were used in the yaml example. I just went and disabled them here so we could get it passing as there is probably not another way around this. You will need to add this line to this page in the package repo, so we don't run into this error again the next time changes are pulled in.